### DISCUSSION TITLE
最近源码编译的固件不能使用zerotier

### DISCUSSION THREAD

**Discussion by lxl6125** [immortalwrt/immortalwrt#752] [Q&A] [answered]
openwrt-18.06-k5.4
x86_64

看着好像是获取不到ipv4地址

ztks55qlxc Link encap:Ethernet  HWaddr 8E:5A:AC:AE:F1:B9  
          inet6 addr: fe80::8c5a:acff:feae:f1b9/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:2800  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:7 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:0 (0.0 B)  TX bytes:746 (746.0 B)

**Comment by 1715173329** [comment]
建议您检查下控制面板是否允许了您的设备接入。

**Comment by lxl6125** [comment]
> 建议您检查下控制面板是否允许了您的设备接入。

控制面板那边显示离线

**Comment by 1715173329** [comment, marked as answer]
建议检查您运营商网络与 zerotier 之间是否能正常通讯。
zerotier-cli listleers

**Comment by lxl6125** [comment]
破案了，运营商的问题
